### ISSUE TITLE
Create NEAR WalletConnect Provider

### ISSUE THREAD

**Issue by trechriron** [near/wallet-selector#924]
### Description

The goal of this story is to build a WalletConnect provider package that’s specific to the NEAR integration similar to [Ethereum](https://github.com/WalletConnect/walletconnect-monorepo/tree/v2.0/providers/ethereum-provider) and [Cosmos](https://github.com/WalletConnect/walletconnect-monorepo/tree/v2.0/providers/cosmos-provider).

I believe the point of these providers (though not terribly clear from documentation) is to give some structure via TypeScript to the generic APIs of the Signer Client such as request to avoid an integration that isn’t actually compliant with the RPC Reference defined in [SQC-157: Create NEAR WalletConnect Provider](https://nearinc.atlassian.net/browse/SQAC-189)

Damir Pjetlovic  August 21, 2023 at 1:06 AM
[Pull request](https://github.com/WalletConnect/walletconnect-monorepo/pull/3465) and [draft PR](https://github.com/WalletConnect/web-examples/pull/270) are raised. We’re waiting for the feedback.

**Comment by kujtimprenkuSQA** [comment]
Updates about Universal Provider:
- NEAR has been added to Universal Providers: https://github.com/WalletConnect/walletconnect-monorepo/pull/3465

Updates about WalletConnect web-examples:
- We Pinged the WalletConnect team again about the example to showcase NEAR with Universal Providers in WalletConnect: https://github.com/WalletConnect/web-examples/pull/270#issuecomment-1744512536
- After some recent refactoring WalletConnect broke the implementation we made a new PR that would fix it: https://github.com/WalletConnect/web-examples/pull/301

**Comment by kujtimprenkuSQA** [comment]
WalletConnect has published a new version that includes NEAR in the Universal Providers: https://github.com/WalletConnect/walletconnect-monorepo/releases/tag/2.10.2

**Comment by kujtimprenkuSQA** [comment]
As mentioned in the above comment https://github.com/near/wallet-selector/issues/924#issuecomment-1752462595 WalletConnect has published the Universal Providers package which includes NEAR provider.

We have also made an extra PR to web-examples of WalletConnect to showcase NEAR with the universal-provider, they have started to refactor their examples but then stopped/postponed it for some reason and we never got any review there: https://github.com/WalletConnect/web-examples/pull/270 

I will be closing this issue as this was originally created to add NEAR to universal-providers, we will see to re-create the example once the refactoring is done on their side.
